app/Http/Controllers/BotController.php

```diff
@@ -73,9 +85,15 @@ public function update(BotUpdateRequest $request, Bot $bot)
         try {
             $bot = $this->botRepository->update($bot->id, $data);
             return redirect()->route('bots.edit', $bot)
-                             ->with('messageSuccess', 'This bot successfully updated');
+                ->with('messageSuccess', [
+                    'status' => 'Update success',
+                    'message' => 'This bot successfully updated'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/BotController.php</br>**Line**: 90</br>Each line in an array declaration must end in a comma